lib/commands/preview.ts

```diff
@@ -0,0 +1,29 @@
+export class PreviewCommand implements ICommand {
+	public allowedParameters: ICommandParameter[] = [];
+
+	constructor(private $liveSyncService: ILiveSyncService,
+		private $projectData: IProjectData,
+		private $options: IOptions,
+		private $previewCommandHelper: IPreviewCommandHelper) { }
+
+	public async execute(args: string[]): Promise<void> {
+		this.$previewCommandHelper.run();
+
+		await this.$liveSyncService.liveSync([], {
+			syncToPreviewApp: true,
+			projectDir: this.$projectData.projectDir,
+			skipWatcher: !this.$options.watch,
+			watchAllFiles: this.$options.syncAllFiles,
+			clean: this.$options.clean,
+			bundle: !!this.$options.bundle,
+			release: this.$options.release,
+			env: this.$options.env,
+			timeout: this.$options.timeout
+		});
+	}
+
+	public async canExecute(args: string[]): Promise<boolean> {
+		return true;
```

Comment:
this will allow the execution of the command even if you write `tns preview someargs`. In case the command does not accept args, just remove the `canExecute` method